src/pages/Restorans/Gallery/Gallery.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Nav from '../../../components/Nav/Nav';
+import HeaderRestoran from '../../../components/Header-Restoran/Header-Restoran';
+import lorem1 from '../../../images/lorem1.png';
+import FooterForRestoran from '../../../components/Footer/FooterForRestoran';
+import leftArrow from '../../../images/left-arrow.svg';
+import rightArrow from '../../../images/right-arrow.svg';
+
+const Gallery = () => {
+  return (
+    <div className='main-galery'>
+      <div className='navigation-menu-for-restoran'>
+        <HeaderRestoran />
+      </div>
+      <div className='nav-and-galery'>
+        <div className='navigation-menu-for-restoran'>
+          <Nav />
+        </div>
+        <div className='container'>
+          <input type='checkbox' name='' id='' className='check' />
+          <ul className='menu-items'>
```

Comment:
`<ul className='menu-items'>` should be removed for now

Reply:
Deleted div with className container